data_updates/Python/iati_rhfp.py

```diff
@@ -5,6 +5,7 @@
 import pandas as pd
 from github import Github, InputGitTreeElement
 
+os.mkdir("iati_csv")
```

Comment:
This will raise a [FileExistsError](https://docs.python.org/3/library/exceptions.html#FileExistsError) error if the folder already exists so you'll need to wrap it in a [`try/except`](https://www.geeksforgeeks.org/python-os-mkdir-method/)

Reply:
I've also rebased your branch so you'll need to pull before making any new changes.

Reply:
cheers Edwin. Added now